src/test/java/com/commercetools/sync/taxcategories/utils/TaxCategorySyncUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package com.commercetools.sync.taxcategories.utils;
```

Comment:
For the testing of `buildActions` method, I would like to see some important use cases to make sure we are covering the updateActions. 
for example, if just name changes, does it create an update action for just ChangeName and also
like with all different values, does updateActions method creates all build actions?

what do you think?

Reply:
I have mixed feelings about this. From one point of view you are totally right, but from another it looks like duplication of tests for TaxCategoryUpdateActionUtils class. So maybe the best solution will be what you proposed - add only one test for a case with all different values and verify if all actions will be built.
Will it be ok?

Reply:
good point, yes it will be enough.